### ISSUE TITLE
metalink and roots

### ISSUE THREAD

**Issue by wyang007** [xrootd/xrootd#1326]
I have a metalink like this:

<?xml version="1.0" encoding="UTF-8"?>
<metalink xmlns="urn:ietf:params:xml:ns:metalink">
  <file name="x">
    <url location="REMOTE" priority="1">root://griddev03.slac.stanford.edu:2094//xrootd/atlas/atlasscratchdisk/rucio/user/fsforza/fa/38/user.fsforza.23139224._000047.hist-output.root</url>
  </file>
</metalink>

xrdcp works with this metalink (the link will be upgraded to TLS)

However, if I replace root:// by roots:// in the metalink, xrdcp complain:

Prepare: [ERROR] Received corrupted data: Malformed or corrupted metalink file.

**Comment by wyang007** [comment]
It looks like github trimmed all the XML tags
```
<?xml version="1.0" encoding="UTF-8"?>
<metalink xmlns="urn:ietf:params:xml:ns:metalink">
  <file name="x">
    <url location="REMOTE" priority="1">root://griddev03.slac.stanford.edu:2094//xrootd/atlas/atlasscratchdisk/rucio/user/fsforza/fa/38/user.fsforza.23139224._000047.hist-output.root</url>
  </file>
</metalink>
```

**Comment by abh3** [comment]
Yes, it looks like this is one that slipped through the cracks. I XrdClMetalinkRedirector.cc:256 the 's' versions of the protocol are not accepted. It should be an easy fix to ass the 's' versions as acceptable protocols.

**Comment by simonmichal** [comment]
Fixed in ad5511d8bd7faa2b50ce2810325aa17fd2c6e0fb.
